primaryHandler.go

```diff
@@ -171,7 +174,7 @@ func authChain(v *viper.Viper, logger log.Logger, registry xmetrics.Registry) (a
 		basculehttp.WithEErrorResponseFunc(listener.OnErrorResponse),
 	)
 
-	constructors := []alice.Constructor{SetLogger(logger), authConstructor, authEnforcer, basculehttp.NewListenerDecorator(listener)}
+	constructors := []alice.Constructor{traceConfig.TraceMiddleware, SetLogger(logger, traceConfig.HeaderConfig), authConstructor, authEnforcer, basculehttp.NewListenerDecorator(listener)}
```

Comment:
It appears that the trace logger is also being added here https://github.com/xmidt-org/scytale/pull/129/files#diff-8d955216dd3a17a0159dcc6ba660370643460686937f540cbc036606e3f63f97R329

Reply:
Here it is added for all the incoming requests we need to log the trace information.